packages/komodo.rb

```diff
@@ -26,12 +22,27 @@ class Komodo < Package
   def self.build
     system "sed -i 's,\$dname/INSTALLDIR/lib/python/bin/python,#{CREW_PREFIX}/bin/python2,' install.sh"
     system "sed -i 's,INSTALLDIR,#{CREW_DEST_PREFIX},' install.sh"
-    system "sed -i '622,630d' support/_install.py"
-    system "sed -i 's,/usr/bin/env python,#{CREW_PREFIX}/bin/env python,' support/_install.py"
-    system "sed -i '61iinstallDir = \"#{CREW_DEST_PREFIX}\"' support/_install.py"
+    system "sed -i '74d' install.sh"
   end
 
   def self.install
-    system './install.sh'
+    system "./install.sh -I #{CREW_DEST_PREFIX}"
+  end
+
+  def self.remove
+    print "Would you like to remove the config directories? [y/N] "
+    response = STDIN.getc
+    config_dirs = ["#{HOME}/.komodoide", "#{HOME}/.activestate"]
+    config_dirs.each { |config_dir|
+      if Dir.exists? config_dir
```

Comment:
This line can be removed as `rm_rf` will not produce error when file not exist

Reply:
True but if the directory doesn't exist then the command is executed unnecessarily.